### ISSUE TITLE
Incompatibility with Socket.io 1.x

### ISSUE THREAD

**Issue by ingro** [scttnlsn/data.io#6]
Hello! I was trying to use your library but got an error on this line (index.js line 5):

```javascript
io.static.add('/data.io.js', { file: path.dirname(__dirname) + '/data.io.js' });
```
*TypeError: Cannot call method 'add' of undefined*

After a little research I saw in your package.json file that data.io dependes on socket.io 0.9.x (maybe you could also write it in your readme).

Do you plan to add supporto for socket.io 1.0?

**Comment by scttnlsn** [comment]
@ingro I just committed some Socket.IO 1.x support on the `socketio` branch.  Would you mind testing it out before I release a new version to NPM?

`npm install git://github.com/scttnlsn/data.io#socketio`

**Comment by ingro** [comment]
Sure, I'll test it out as soon as I can!

Thanks for your fast response :)

**Comment by ingro** [comment]
Just to report that I tried your Backbone's example and all is working good, only thing I needed to change was the scr for data.io.js that don't need the socket.io prefix anymore:

```html
<script src="/data.io.js"></script>
```

Good work! :+1:

**Comment by scttnlsn** [comment]
@ingro That should have already been changed in the example (https://github.com/scttnlsn/data.io/commit/cc4605c4889a80ff615b08cc49500fc00956ae16#diff-04c6e90faac2675aa89e2176d2eec7d8R46).  Did you need to update this path somewhere else as well?

Thanks!

**Comment by ingro** [comment]
Nope sorry, I was looking at the master version!

**Comment by scttnlsn** [comment]
Available in v0.3.0
